swap_meet/vendor.py

```diff
@@ -1,2 +1,84 @@
+import uuid
+
+#from item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=[]):
+        self.inventory = inventory #each vendor will have an attribute named :inventory (an empty list)
```

Comment:
This is what's failing your integration tests. In Python, when you assign a "mutable" object like a list as a default value, *every time the function uses that default value* it will share the same instance of that object. The workaround is to use `None` as a default for lists and check for `None` in the initializer:

```python
    def __init__(self, inventory = None):
        self.inventory = inventory if inventory else []
```

^-- if you do this instead, your integration tests will pass (after you unskip them).